src/Parser/MistweaverMonk/CombatLogParser.js

```diff
@@ -597,6 +625,58 @@ class CombatLogParser extends MainCombatLogParser {
           </dfn>
         ),
       },
+      this.modules.archiveoffaith.active && {
+        id: ITEMS.ARCHIVE_OF_FAITH.id,
+        icon: <ItemIcon id={ITEMS.ARCHIVE_OF_FAITH.id} />,
+        title: <ItemLink id={ITEMS.ARCHIVE_OF_FAITH.id} />,
+        result: (
+          <span>
+            <dfn data-tip="The actual effective healing contributed by the Archive of Faith on-use effect.">
```

Comment:
Would it be sufficient to show the total healing here and the breakdown (dot and shield proc) in the tooltip instead?